data/mods/Magiclysm/bionics.json

```
@@ -6,10 +6,35 @@
     "description": "Embedded into your back and connected to your bionic power supply is a powerful spell focus able to consume blood from dead creatures to produce bionic power.  It can store up to 100 mL of blood.",
     "occupied_bodyparts": [ [ "torso", 8 ] ],
     "fuel_options": [ "blood", "dragon_blood", "tainted_blood", "hblood" ],
-    "fuel_capacity": 100,
     "fuel_efficiency": 0.6,
     "exothermic_power_gen": true,
     "time": "6 s",
-    "flags": [ "BIONIC_TOGGLED", "BIONIC_POWER_SOURCE" ]
+    "flags": [ "BIONIC_TOGGLED", "BIONIC_POWER_SOURCE" ],
+    "passive_pseudo_items": [ "integrated_blod_bank" ]
+  },
+  {
+    "id": "integrated_blod_bank",
```

```suggestion
    "id": "integrated_blood_bank",
```
Spelling is not critical but this might spare some confusion